cmd/sriovdp/manager.go

```diff
@@ -100,13 +150,85 @@ func (rm *resourceManager) readConfig() error {
 	return nil
 }
 
-func (rm *resourceManager) initServers() error {
+// StoreConfig saves SR-IOV Device Plugin config into the file
+func (rm *ResourceManager) StoreConfig() error {
```

Comment:
why do we need to store config on host ?

Reply:
agree, it should be deleted